NEWS

```diff
@@ -1,3 +1,13 @@
+* Version 1.2.11 (unreleased)
+- Added Namespace support (#30). radcli now supports Namespace on 
+  Linux based systems. 
+
+
+* Version 1.2.10 (unreleased)
```

Comment:
version 1.2.10 was merged after I pulled my branch. Hence, added 1.2.10 manually to resolve conflicts.